src/jumio-kyc/fixtures/workflow.ts

```diff
@@ -14,6 +15,7 @@ export const WORKFLOW_RETRIEVE_FIXTURE = (
   decisionStatus: DecisionStatus,
   userId = '1',
   imageCheck: ImageCheck = IMAGE_CHECK_FIXTURE,
```

Comment:
In another PR I am gonna make this input an object instead of parameters, it is getting too big.